### PR TITLE
Add an image list command, and make features order deterministic

### PR DESCRIPTION
**Pull request by redref**

### CODE REVIEW

File: pkr/cli/parser.py [hunk 1]
```diff
@@ -56,6 +56,15 @@ def _pull_images(args):
     kard.docker_cli.pull_images(services, registry, tag=args.tag)
 
 
+def _list_images(args):
+    kard = Kard.load_current()
+    services = args.services or list(kard.env.get_container().keys())
+    if args.tag is None:
+        args.tag = kard.meta['tag']
+    for service in services:
+        print(kard.docker_cli.make_image_name(service, args.tag))
```
Comment (Kimamisa) [inline pkr/cli/parser.py:65]: use `write` for printing text

File: tox.ini [hunk 2]
```diff
@@ -10,7 +10,7 @@ commands = pytest test --junit-xml pkr-test-report-{envname}.xml --junit-prefix
 
 [testenv:pylint]
 deps = pylint
-commands = bash -c \'{envbindir}/pylint -f parseable -r no --py3k {toxinidir}/pkr\'
+commands = pylint -f parseable -r no --py3k {toxinidir}/pkr
```
Comment (Kimamisa) [inline tox.ini:13]: removing this breaks on some envs

File: pkr/environment.py [hunk 3]
```diff
@@ -27,9 +27,8 @@ def __init__(self, env_name, features=None, path=None):
 
         self.env = self._load_env_file(env_file_path)
 
-        self.features = features or []
-        self.features.extend(self.env.get('default_features', []))
-        self.features = list(set(self.features))
```
Comment (Kimamisa) [inline pkr/environment.py:32]: this prevents having duplicates in the meta file

File: tox.ini (hunk 2 above)
Comment (redref) [reply tox.ini:13]: Actually, it is broken for me with this. Not sure why, launching tox from inidir.

Get : 
```
pkr\:1: [F0001(fatal), ] No module named pkr\
```

**Pull request merged**